src/rating/styled-components.js

```diff
@@ -57,15 +56,17 @@ export const StyledStar = styled<StyledRatingItemPropsT>(
       marginTop: 0,
       marginBottom: 0,
       marginRight: $theme.sizing.scale300,
-      width: '22px',
-      height: '20px',
+      width: $size,
```

Comment:
The above it says that the type for the `size` is specified as `number` therefore we'd need to add `px` here if type is `number`.